package.json

```diff
@@ -14,17 +14,32 @@
   "main": "./bin/cli.js",
   "scripts": {
     "lint": "eslint ./lib",
-    "install-commit-validator": "fit-commit-js install"
+    "install-commit-validator": "fit-commit-js install",
+    "test": "jest"
```

Comment:
I think we should also lint the tests, when we don't do it, the coding style of the tests will not be consistent with the style of the code under /lib